packages/main/src/themes/Table.css

```diff
@@ -39,4 +39,8 @@
     inset: 0;
     height: 100%;
     z-index: 2;
+}
+
+#loading[_is-busy] {

```

Comment:
@nnaydenow shouldn't this be handled by the busy indicator itself?
it seems like the following lines expects busy indicator to be rendered as parent of the component but we render it as a child of the table. If you guys wants change it please let me know the I can remove it. I am not happy writing a style for the private attribute of another component.
https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/themes/BusyIndicator.css#L98-L100

Reply:
+1
Also not a fan of this, it would be good if we could avoid this somehow.

Is is not feasible to check BusyIndicator.active instead? Or are there issues with rendering then?

Reply:
I simply expect BusyIndicator adds its own dim background color, independent from whether it is rendered as a child or a wrapper.